node/Cargo.toml

```
@@ -38,4 +38,5 @@ hex = "0.4.3"
 num-bigint = "0.4.6"
 transitive = "1.1.0"
 secp256k1 = "0.30.0"
-libp2p = { version = "0.55.0", features = ["identify", "quic", "tokio","ping","macros"] }
+libp2p = { version = "0.55.0", features = ["identify", "quic", "tokio","ping","macros","kad","dns"] }
```

We don't have any formatting for `toml`.
```suggestion
libp2p = { version = "0.55.0", features = ["identify", "quic", "tokio", "ping", "macros", "kad", "dns"] }
```
Can you commit this?